Give class names a prefix on reference pages

wong-justin opened the pull request:
fixes #650

Before: blur reference page would have .blur CSS style applied 
After: blur reference page has .reference_blur class name, so there are no more collisions with CSS styles

limzykenneth commented on src/data/examples/zh-Hans/08_Math/10_Interpolate.js:
```diff
@@ -1,10 +1,8 @@
- * the lerp() function.
+ * @description 在屏幕上移动鼠标，圆会跟着移动。
+ * 在动画绘制的每一帧之间，lerp() 函数会使圆从其当前位置向光标移动一部分距离（0.05）。
  * This is the same as the Easing under input only with lerp() instead..
```
It is refering to the "Easing" [example](https://p5js.org/examples/input-easing.html) under the "Input" section.

MuyingLi replied:
How should we translate "Easing" or just keep it, it's also for later translation in Easing example? I kept it in my translation.

limzykenneth replied:
It's up to you, as you noted, if you translate that example later on it should be consistent here as well so a bit of future planning.

MuyingLi commented on src/data/examples/zh-Hans/08_Math/06_sinewave.js:
```diff
@@ -1,16 +1,16 @@
- * @description Render a simple sine wave.
- * Original by Daniel Shiffman.
+ * @name 正弦波
+ * @description Render 是一个简单的正弦波。
```
Sure. I'll put "渲染" as its translation. But just curious, in this situation, isn't "render" same as "create"

limzykenneth replied:
It is used in similar way but I think in this case it's just what the original Processing example used so we just copy it over more or less.

MuyingLi commented on src/data/examples/zh-Hans/08_Math/07_additivewave.js:
```diff
@@ -38,20 +36,19 @@ function draw() {
     let x = theta;
     for (let i = 0; i < yvalues.length; i++) {
-      // Every other wave is cosine instead of sine
+      // 每一个其他的波是余弦，而不是正弦
```
I re-translated it.

limzykenneth replied:
`// 接下来的所有波都是余弦，而非正弦`

That's not quite what it means. The idea is that `0` is cosine, `1` is sine, `2` is cosine, `3` is sine and so on. I can't immediately think of an expression in Chinese that conveys that, maybe you can come up with something.

MuyingLi replied:
I could not find a good way to translate it...  @sm7515 can you help me with this?

sm7515 replied:
maybe something like "每隔一个余弦之后是正弦" or “正弦余弦交替”

limzykenneth commented:
Looks great! Thanks for all your effort.

wong-justin committed:
give class names a prefix on reference pages

fixes #650
before: blur reference page would have .blur CSS style applied
after: blur reference page has .reference_blur class name,
so there are no more collisions with CSS styles

hopefully this doesn't have any unintended side effects

limzykenneth commented:
Looks good. Thanks!
